spk/rutorrent/src/installer.sh

```diff
@@ -46,26 +49,40 @@ syno_group_remove ()
     fi
 }
 
-set_syno_permissions ()
+set_traverse_permissions () 
```

Comment:
This old beta code has been replaced by generic installer to avoid copy-paste pattern: https://github.com/SynoCommunity/spksrc/blob/master/mk/spksrc.service.installer#L145
Really it is worth move to generic installer:

Reply:
I moved to the generic installer. 

By doing so I found a tiny bug though in the (very much used) `set_syno_permissions` function and I find this a little bit unsettling and weird... :/

Reply:
My bad about the tiny bug I had misunderstood the way it worked. A classic.